instrumentation/httpx/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History: opentelemetry-instrumentation-http
+
```

Comment:
The Release automation will generate the Changelog for you. It is very strict and will not allow manual edits to this file. 

```suggestion
```